pyqtgraph/functions.py

```diff
@@ -152,15 +206,14 @@ def siFormat(x, precision=3, suffix='', space=True, error=None, minVal=1e-25, al
         else:
             plusminus = " +/- "
         fmt = "%." + str(precision) + "g%s%s%s%s"
-        return fmt % (x*p, pref, suffix, plusminus, siFormat(error, precision=precision, suffix=suffix, space=space, minVal=minVal))
+        return fmt % (x*p, pref, suffix, plusminus, siFormat(error, precision=precision, suffix=suffix, space=space, minVal=minVal, power=power))
 
 
 def siParse(s, regex=FLOAT_REGEX, suffix=None):
     """Convert a value written in SI notation to a tuple (number, si_prefix, suffix).
 
-    Example::
-
-        siParse('100 µV")  # returns ('100', 'µ', 'V')
+    Example:
```

Comment:
The original formatting with `::` and the extra blank line are needed for sphinx to render the documentation correctly (see: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)